README.md

```diff
@@ -20,6 +20,10 @@ limitations under the License.
 Windows is not an explicitly supported environment, although where possible Palisade has been made compatible.  
 For Windows developer environments, we recommend setting up [WSL](https://docs.microsoft.com/en-us/windows/wsl/).
 
+### Getting Started
+This is not the starting point for understanding Palisade.
```

Comment:
I went for another tack and expanded on what is at the other end of the link "For an overview of Palisade, start with the Palisade introduction and the accompanying guides: QuickStart Guide; and Developer Guide:"

Reply:
I'm not sure what the best way of describing this is. I don't think it needs to be too descriptive, just a pointer to the main repo

Reply:
I dont think it matters too much, so am happy to leave this as is